server/plugin/command_test.go

```diff
@@ -883,8 +897,35 @@ func TestHandleSubscriptionsList(t *testing.T) {
 				assert.Equal(t, expected, result)
 			},
 		},
+		{
+			name:      "Subscriptions with flags",
+			channelID: "channel4",
+			setup: func() {
+				mockKvStore.EXPECT().Get(SubscriptionsKey, gomock.Any()).DoAndReturn(func(key string, value **Subscriptions) error {
+					*value = &Subscriptions{
+						Repositories: map[string][]*Subscription{
```

Comment:
Create util if possible as mentioned in previous PR https://github.com/mattermost/mattermost-plugin-github/pull/846#discussion_r1891561185